btas/generic/tucker.h

```diff
@@ -92,5 +92,118 @@ namespace btas {
 #endif  // BTAS_HAS_INTEL_MKL
     }
   }
+
+  template<typename Tensor>
+  void make_tucker_factors(Tensor& A, double epsilon_svd,
```

Comment:
`const Tensor& A`?

Reply:
This is not constant because the method has an option to convert the tensor `A` into the Tucker core tensor.